global-api/importer/datasource_seeder/edgar_seeder.py

```diff
@@ -0,0 +1,107 @@
+"""seed datasource table with edgar data
+-- query to get start and end year
+SELECT
+DISTINCT
+    reference_number,
+    MIN(year) AS start_year,
+    MAX(year) AS end_year
+FROM "GridCellEmissionsEdgar"
+GROUP BY "reference_number";
+"""
+import argparse
+import os
+from sqlalchemy import create_engine, MetaData, Table
+from utils import DataSource, uuid_generate_v3, insert_record
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--database_uri",
+        help="database URI (e.g. postgresql://ccglobal:@localhost/ccglobal)",
+        default=os.environ.get("DB_URI"),
+    )
+    args = parser.parse_args()
+
+    engine = create_engine(args.database_uri)
+    metadata_obj = MetaData()
+    table = Table("datasource", metadata_obj, autoload_with=engine)
+
+    BASE_URL = "https://ccglobal.openearth.dev"
+    PUBLISHER = "Joint Research Centre"
+
+    # ---------------------------------------
+    # create datasource objects
+    # ---------------------------------------
+
+    # I.3.1
+    datasource_data = dict(
+        publisher_id=PUBLISHER,
+        name="EDGARv7.0 Combustion for manufacturing",
+        description="",
+        source_type="",
+        access_type="globalapi",
+        url="https://joint-research-centre.ec.europa.eu/index_en",
+        geographical_location="global",
+        start_year="2021",
+        end_year="2021",
+        lastest_accounting_year="2021",
+        frequency_of_update="",
```

Comment:
This would be good to add, so if it's monthly, quarterly, yearly etc.

Reply:
I put the frequency of update to "annual". I'm pretty sure that is right for EDGAR since they release their reports annually. I remember in a meeting with ClimateTRACE they said they planned on doing annual releases for most assets.